### ISSUE TITLE
project rename, resource path is still under old project name

### ISSUE THREAD

**Issue by whysqwhw** [angelozerr/tern.java#421]
Eclipse content assist set auto and prompt more characters.

egg.
<link href="css/base.css">

It will prompt "href" and it's value. The former seems supported by jsdt and the latter is by angularjs-eclipse.It's cool.But after I changed the project name, it does not work anymore unless I restart Eclipse.
In Eclipse's "Error Log" view there are some resource path errors which are use old project name.

I find some rename related plugins as follows.But I do not how it works.I guess tern.java need do something,so I 
report a issue here.

org.eclipse.ltk.core.refactoring
org.eclipse.ltk.ui.refactoring
org.eclipse.jdt.ui
org.eclipse.wst.jsdt.ui

**Comment by angelozerr** [comment]
Could you please give a simple and detailled scenario with your problem, please.

**Comment by whysqwhw** [comment]
1.Create a new javascript project which named "HelloWorld". In this project, it contains some css files in a folder. The index.html will link the css files.
2. Input the `<link>` tag, it will prompt the css files path.
3. Rename the project's name as "HH2".
4. The `<link>` tag cannot prompt the css files path.
5. Then open the Error Log view of eclipse, there are multiple errors. It shows "Error while collecting resource for container css". Resource '/HelloWorld/css/base.css' does not exist.
6. The project name is already "HH2", but the eclipse still tries to find the old path. Perhaps the project's context does not change.

**Comment by angelozerr** [comment]
This issue is not for tern.java, but for https://github.com/angelozerr/eclipse-wtp-webresources

But I think I will give up this project since @mickaelistria is to move features from  https://github.com/angelozerr/eclipse-wtp-webresources to WTP HTML Editor step by step.

**Comment by whysqwhw** [comment]
Yes,hello cow force. I fix it by change WebResourcesIndexManager.performAction function  to update WebResourcesProjectConfiguration data; and WebResourcesProviderType.collect function to avoid resource not exists.Is it ok?

**Comment by mickaelistria** [comment]
@whysqwhw Would you rather be interested in contributing those features to Eclipse directly? As @angelozerr explained, we're trying to move/copy those features to Eclipse WTP directly (where the HTML editor lives). See https://bugs.eclipse.org/bugs/show_bug.cgi?id=495132 for example.

**Comment by whysqwhw** [comment]
ok,thanks

**Comment by mickaelistria** [comment]
@whysqwhw a nice contribution would be to try the Gerrit patch I suggested and to report whether it works for your use-case or not ;)
